tests/unit/versioned/test.tap.js

```diff
@@ -342,6 +342,22 @@ tap.test('Will not filter tests when keywords are an empty list', function (t) {
   t.end()
 })
 
+tap.test('should filter based on multiple keywords', function (t) {
+  const test = new Test(
+    MOCK_TEST_DIR,
+    {
+      bluebird: ['1.0.8', '1.1.1', '1.2.4', '2.0.7'],
+      redis: ['1.0.0', '2.0.1', '2.1.0']
+    },
+    {
+      testPatterns: ['other.mock.js', 'redis']
+    }
+  )
+
+  t.equal(test.matrix._matrix[1].tests.files.length, 2, 'should include only one test file')
```

Comment:
This says 'should include only **one** test file' but asserts against a length of 2.